### PR TITLE
fix: support post type as array

### PR DESCRIPTION
**Pull request by codepuncher**

### CODE REVIEW

**Review by danlapteacru — APPROVED** [review]

File: src/Settings/PostGrid.php [hunk 1]
```diff
@@ -189,15 +189,17 @@ public static function filterBar($settings): string
...
+        $postTypes = (array) ($settings->post_type ?? ['post']);
         if ('main_query' === $settings->data_source) {
-            $postType = get_post_type();
+            $postTypes = [get_post_type()];
         }
-        if (empty($postType)) {
-            $postType = 'post';
+        if (empty($postTypes)) {
+            $postTypes = ['post'];
         }
```
Comment (TangRufus) [inline src/Settings/PostGrid.php:198]: `get_post_type()` can return `false`.

```shell
php > var_dump(empty([false]));
bool(false)
```

Comment (danlapteacru) [reply src/Settings/PostGrid.php:198]: Rufua is alive 😲

Comment (codepuncher) [reply src/Settings/PostGrid.php:198]: Good thing I check for false :) hello pal!

Comment (TangRufus) [reply src/Settings/PostGrid.php:198]: Oh, i see line 202 now.
I am stupid as always

Comment (codepuncher) [reply src/Settings/PostGrid.php:198]: `reset` can also return `false` :) double-check!

hope you are doing well mate

**Pull request merged**